packages/renderer/src/lib/dashboard/ReleaseNotesBox.svelte

```diff
@@ -0,0 +1,94 @@
+<script lang="ts">
+import { faCircleArrowUp } from '@fortawesome/free-solid-svg-icons';
+import { Button, CloseButton, Link } from '@podman-desktop/ui-svelte';
+import { onDestroy, onMount } from 'svelte';
+
+import type { ReleaseNotes } from '/@api/release-notes-info';
+
+import Markdown from '../markdown/Markdown.svelte';
+
+let showBanner = false;
+let notesAvailable = false;
+let updateAvilable = false;
+let notesURL: string;
+let currentVersion: string;
+let notesInfo: ReleaseNotes | undefined;
+const receiveShowReleaseNotes = window.events?.receive('show-release-notes', () => {
+  showBanner = true;
+});
+function openReleaseNotes() {
+  window.openExternal(notesURL);
+}
+function updatePodmanDesktop() {
+  window.updatePodmanDesktop();
+}
+async function getInfoFromNotes() {
+  let getNotes = await window.podmanDesktopGetReleaseNotes();
+  notesAvailable = getNotes.releaseNotesAvailable;
+  if (notesAvailable) {
+    notesInfo = getNotes.notes;
+  }
+  notesURL = getNotes.notesURL;
+}
+function onClose() {
+  window.updateConfigurationValue(`releaseNotesBanner.show`, currentVersion);
+  showBanner = false;
+}
+onMount(async () => {
+  currentVersion = await window.getPodmanDesktopVersion();
+  showBanner = (await window.getConfigurationValue(`releaseNotesBanner.show`)) !== currentVersion ? true : false;
+  window
+    .podmanDesktopUpdateAvailable()
+    .then(available => (updateAvilable = available))
+    .catch(() => {
+      console.log('Cannot check for update');
+    });
```

Comment:
wondering here why it's using the pattern promise.then while for the previous or after lines you're using await promise